src/promptflow-tools/promptflow/tools/yamls/llm.yaml

```diff
@@ -218,6 +218,7 @@ promptflow.tools.llm.llm:
     tool_choice:
       type:
       - object
```

Comment:
add comment to explain why 'object' must be the first type.

please be nicer to other maintainers.

Reply:
comment added, please review